okio-fakefilesystem/build.gradle

```diff
@@ -44,6 +44,13 @@ kotlin {
   }
 }
 
+tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
```

Comment:
nit: move this block to the top-level build.gradle under the `subprojects` block?

Reply:
Good idea.